routes/server.go

```diff
@@ -1044,6 +1045,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.GetUserDerivedKeys,
 			PublicAccess,
 		},
+		{
+			"GetSingleDerivedKey",
+			[]string{"GET"},
+			RoutePathGetSingleDerivedKey + "/{ownerPublicKeyBase58Check:[0-9a-zA-Z]{54,55}}/{derivedPublicKeyBase58Check:[0-9a-zA-Z]{54,55}}",
```

Comment:
What does the regex mean here? Is that a type of validation guard?

Reply:
it just ensures that we get a valid public key at this route param